travis/cfg/travis_run_pylint_beta.cfg

```diff
@@ -14,23 +14,28 @@
 #   duplicate-xml-fields - W7907
 #   duplicate-xml-record-id - W7902
 #   incoherent-interpreter-exec-perm - W8201
+#   invalid-commit - E8102
 #   javascript-lint - W7903
 #   manifest-author-string - E8101
 #   manifest-deprecated-key - C8103
 #   manifest-required-author - C8101
 #   manifest-required-key - C8102
+#   method-compute - C8108
+#   method-inverse - C8110
```

Comment:
This convention is not explicit on the [guidelines](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#field). It should be added there.